README.md

```diff
@@ -135,11 +144,11 @@ They can also be used instead of a configuration file.
   KIOSK_WINDOW_POSITION string
       Top Left Position of Kiosk (default "0,0")
   KIOSK_IGNORE_CERTIFICATE_ERRORS bool
-      ignore SSL/TLS certificate errors (default "false")
+      Ignore SSL/TLS certificate errors (default "false")
   KIOSK_IS_PLAYLIST bool
       URL is a playlist (default "false")
   KIOSK_LOGIN_METHOD string
-      [anon|local|gcom|goauth|idtoken|apikey] (default "anon")
+      [anon|local|gcom|goauth|idtoken] (default "anon")
```

Comment:
Why do you remove the `apikey` value from the doc?

Reply:
my mistake - merge failure, will add it back, thanks for the catch!

Reply:
i took the output from the command-line help output, looks like the option wasn't added to the env section. i've added it back in. thanks again!